index.html

```diff
@@ -371,6 +371,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="https://ericrosenbaum.github.io/spotify-extension/" target="_blank">
```

Comment:
I made a note on this above. I think in the current design (with the main link direct to the editor with the extension loaded), the likely scenario is that the user never sees the documentation (de-emphasized as a small link below), and probably only ever sees a single sample project (the one below, as opposed to any linked from the docs), if any. This design is reasonable for, for example, the twitter or speech extensions, where the docs and examples are pretty minimal and the blocks are pretty self-explanatory. For the Spotify extension, which has a video, six examples, block documentation with code samples, etc., the docs page really should be the main entry point.    

I guess if we're concerned with consistency then I would argue for making all of them point to the docs. In place of the "documentation" link we could put a link that opens a blank project with the extension loaded (need to figure out the wording for that).